ocaml/libs/xapi-stdext/lib/xapi-stdext-date/dune

```diff
@@ -1,16 +1,6 @@
 (library
   (name xapi_stdext_date)
   (public_name xapi-stdext-date)
-  (modules :standard \ test)
-  (libraries astring
-             ptime
-             ptime.clock.os
-             unix)
-)
-
-(test
-  (name test)
-  (package xapi-stdext-date)
-  (modules test)
-  (libraries alcotest xapi-stdext-date ptime)
+  (modules :standard)
+  (libraries clock)
```

Comment:
ptime is still required, I get this failure on my branch with transitive deps turned off:
File "ocaml/libs/xapi-stdext/lib/xapi-stdext-date/date.mli", line 20, characters 15-22:
20 | val of_ptime : Ptime.t -> t
                    ^^^^^^^
Error: Unbound module Ptime

This is because date.mli declares a dependency on Ptime (without going through clock)

Reply:
```
   (modules :standard)
-  (libraries clock)
+  (libraries clock ptime)
```

Reply:
I think re-exporting is fair